tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_last_seen_v1/test_day_bit_shifting/expect.yaml

```diff
@@ -23,17 +23,23 @@
   total_searches: 0
   tagged_searches: 0
   engine_searches:
-  - value:
-      total_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      tagged_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      search_with_ads: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      ad_click: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-  days_seen_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABw==
-  days_searched_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
-  days_tagged_searched_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
-  days_searched_with_ads_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
-  days_clicked_ads_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
-  days_created_profile_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABkA==
+    - value:
+        total_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        tagged_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        search_with_ads: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        ad_click: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+  days_seen_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+   AAAAAAAAAAAAAAAAAABw==
```

Comment:
There are a few tests that are failing because the newline here introduces a trailing whitespace:

```
    'days_tagged_searched_bytes': 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA '
                                  'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==',
```

Please trim any trailing whitespace.

Reply:
Trimming the trailing space causes errors in a number of files

Reply:
With `allow-non-breakable-inline-mappings: true`, you should be able to leave this in one piece.